presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -170,15 +170,22 @@ public static ColumnMapping decimalColumnMapping(DecimalType decimalType, Roundi
         if (decimalType.isShort()) {
             return ColumnMapping.longMapping(
                     decimalType,
-                    (resultSet, columnIndex) -> encodeShortScaledValue(resultSet.getBigDecimal(columnIndex), scale),
+                    shortDecimalReadFunction(decimalType, UNNECESSARY),
```

Comment:
There should be a comment about why long and short decimal use different rounding.

Reply:
Yes, and actually i also wondered why. It's preexisting and i maintained this.
This was introduced in https://github.com/prestosql/presto/pull/2088, so maybe @guyco33 or @kokosing can suggest the comment wording to be added here

Reply:
added `// TODO why roundingMode is not used here, but it is used for long decimal only?` comment for now, but of course this is not ideal

Reply:
I found that mapping to short decimal doesn't need rounding, since its max precision (less then 18) always fit and unspecified decimal (special to Postgres) is mapped to long decimal

Reply:
replaced with a check